### DISCUSSION TITLE
DOuble scrolling with tui-scroll-controls?

### DISCUSSION THREAD

**Discussion by vytautas-pranskunas-** [taiga-family/taiga-ui#9359] [Q&A] [answered]
Do you have any idea why do i get double scrolling when my main content is 100% (or 100vh) height? it suppose to end with paging... 
![Screenshot 2024-10-06 at 11 14 36](https://github.com/user-attachments/assets/e2991cef-71d8-44d4-afa7-66aabe8b9046)

p.s. higher the overflow area - higher double scrolling area..

my html markup was this

```
<div class="flex"> <!-- when this is 100vh then problem exists -->
    <pa-side-menu /> <!-- when this is 100vh then problem DOES NOT exists -->
    <div class="flex column"> 
        <pa-header />
        <div class="content">
            <router-outlet /> <!-- this makes a large scrollable list (overflow) -->
        </div>
    </div>
</div>
```

**Comment by waterplea** [comment, marked as answer]
You would have to investigate that in devtools yourself, Taiga UI global scrollbars only reflect the overflow status of `html` tag and do not do anything else on their own, something in your layout is causing this.
